.gitignore

```diff
@@ -1 +1,4 @@
-/data
\ No newline at end of file
+/data
+.vscode
+app
+cmd/ddns-updater/data/
```

Comment:
Why would the data directory be in `cmd/ddns-updater/data/`?
On my side, I usually build and run from the root of the workspace with `go run ./cmd/ddns-updater/main.go` and it picks up the `data` directory there.

Reply:
It should be ignored because while developing you would probably add your ddns secrets in there which would then be leaked if pushed